target/linux/generic/files/drivers/mtd/mtdsplit/mtdsplit_uimage.c

```diff
@@ -321,6 +321,55 @@ static struct mtd_part_parser uimage_netgear_parser = {
 	.type = MTD_PARSER_TYPE_FIRMWARE,
 };
 
+
+/**************************************************
+ * ALLNET
+ **************************************************/
+
+#define FW_MAGIC_SG8208M	0x00000006
+#define FW_MAGIC_SG8310PM	0x83000006
+
+static ssize_t uimage_verify_allnet(u_char *buf, size_t len, int *extralen)
+{
+	struct uimage_header *header = (struct uimage_header *)buf;
+
+	switch (be32_to_cpu(header->ih_magic)) {
+	case FW_MAGIC_SG8208M:
+	case FW_MAGIC_SG8310PM:
```

Comment:
Do you know if this format was invented by allnet or is this from Realtek? If this is seen on different Realtek switches, you should probably rename this to Realtek.

Reply:
No, these 2 cases from Allnet are the only time I have seen it for these types of Realtek switches, all the others use standard denx, apart from Netgear who are using their own format as usual.